### PR TITLE
Qa 77

### PR DESCRIPTION
**Pull request by oleorhagen**
The updated client has not yet been tested. Still it would be nice if you had a look :)

### CODE REVIEW

File: docker_push [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker push USER/REPO
```
Comment (kacf) [inline docker_push]: Copy paste error? `USER/REPO` looks like the example string, not what you want to push.

Comment (oleorhagen) [reply docker_push]: Yes, that is correct. What is the repo?

Comment (mzedel) [reply docker_push]: that would be mendersoftware/mender-stress-test-client - so far this doesn't exist, but would probably be created on first push

**Comment by oleorhagen** [comment]
So I have run it locally with the demo environment, and 10 clients and it works. Are there any other tests you want done?

**Comment by oleorhagen** [comment]
Also, I guess there is no way I can test the deployment?

File: main.go [hunk 2]
```diff
@@ -204,6 +205,19 @@ func clientAuthenticate(c *client.ApiClient, storeFile string) client.AuthToken
 	}
 }
 
+func stressTestClientServerIterator() func() *client.MenderServer {
+	serverIteratorFlipper := true
+	return func() *client.MenderServer {
+		// Only empty the iterator after five attempts
```
Comment (kacf) [inline main.go]: Does the comment match what actually happens? Looks like it's binary flipping.

Comment (oleorhagen) [reply main.go]: Nice, thank you!

**Comment by oleorhagen** [comment]
@mzedel , this is final, and tested locally. You happy?

**Pull request merged**